lib/ProductOpener/Display.pm

```diff
@@ -6363,145 +6320,21 @@ HTML
-		$image_banner = <<HTML
-<div id="donate_banner" class="row">
-<div class="small-12 large-12 xlarge-8 xxlarge-7 columns">
-<div id="image_banner" style="margin-bottom:1rem;" style="display:none;"><a href="$link?utm_source=off&utm_medium=web&utm_campaign=donate-2019&utm_term=$banner"><img src="$image" alt="" /></a></div>
-<div><input id=\"hide_image_banner\" type=\"checkbox\"><label for=\"hide_image_banner\">
-<span id="hide_image_banner_hide" style="display:none;">Ho già donato o non sono interessato. Nascondi il banner.</span>
-<span id="hide_image_banner_show" style="display:none;">Ehi, una casella da deselezionare?</span>
-</label></div>
-</div>
-</div>
-HTML
-;
-	}
-
-	elsif ($lc eq 'de') {
-
-		my $link = lang("donate_link");
-		my $image;
-		my $utm;
-		my @banners = qw(independent personal research);
-		my $banner = $banners[time() % @banners];
-		$image = "/images/banners/donate/donate-banner.$banner.$lc.800x150.svg";
-		$image_banner = <<HTML
-<div id="donate_banner" class="row">
-<div class="small-12 large-12 xlarge-8 xxlarge-7 columns">
-<div id="image_banner" style="margin-bottom:1rem;" style="display:none;"><a href="$link?utm_source=off&utm_medium=web&utm_campaign=donate-2019&utm_term=$banner"><img src="$image" alt="" /></a></div>
-<div><input id=\"hide_image_banner\" type=\"checkbox\"><label for=\"hide_image_banner\">
-<span id="hide_image_banner_hide" style="display:none;">Ich habe bereits gespendet oder ich bin nicht interessiert. Banner ausblenden.</span>
-<span id="hide_image_banner_show" style="display:none;">Hey, eine Box zum abwählen?!</span>
-</label></div>
-</div>
-</div>
-HTML
-;
-	}
-
-	# Display the English banner if we don't have another one.
-	elsif (1 or ($lc eq 'en')) {
-
-		my $link = lang("donate_link");
-		my $image;
-		my $utm;
-		my @banners = qw(independent personal research);
-		my $banner = $banners[time() % @banners];
-		$image = "/images/banners/donate/donate-banner.$banner.en.800x150.svg";
-		$image_banner = <<HTML
-<div id="donate_banner" class="row">
-<div class="small-12 large-12 xlarge-8 xxlarge-7 columns">
-<div id="image_banner" style="margin-bottom:1rem;" style="display:none;"><a href="$link?utm_source=off&utm_medium=web&utm_campaign=donate-2019&utm_term=$banner"><img src="$image" alt="" /></a></div>
-<div><input id=\"hide_image_banner\" type=\"checkbox\"><label for=\"hide_image_banner\">
-<span id="hide_image_banner_hide" style="display:none;">$Lang{donation_banner_hide}{$lc}</span>
-<span id="hide_image_banner_show" style="display:none;">$Lang{donation_banner_show}{$lc}</span>
-</label></div>
-</div>
-</div>
-HTML
-;
-	}
-
-	if ($server_options{producers_platform}) {
-		$image_banner = "";
-	}
-
-	if ($image_banner ne "") {
+	my $link = lang("donate_link");
+	my $image;
+	my $utm;
+	my @banners = qw(independent personal research);
+	my $banner = $banners[time() % @banners];
+	$image = "/images/banners/donate/donate-banner.$banner.$lc.800x150.svg";
```

Comment:
For some languages, this file will not exist. We need to check if the file exists (with -e "$www_root//images/banners/donate/donate-banner.$banner.$lc.800x150.svg")  --> if it does not exist, we need to use the English banner.